packages/plugins/rubygem-foreman_webhooks/rubygem-foreman_webhooks.spec

```diff
@@ -89,26 +69,27 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/config
 %{gem_instdir}/db
 %{gem_libdir}
-%exclude %{gem_cache}
 %exclude %{gem_instdir}/package.json
 %exclude %{gem_instdir}/webpack
-%exclude %{gem_instdir}/Rakefile
+%exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_plugin}
-%{foreman_assets_plugin}
-%{foreman_assets_foreman}
```

Comment:
Also, same question: is this supposed to be removed?..

Reply:
No, not at all.

It fails to build exactly because of these:
```
error: Installed (but unpackaged) file(s) found:
   /usr/share/foreman/public/assets/foreman_webhooks
   /usr/share/gems/gems/foreman_webhooks-3.1.0/public/assets/foreman_webhooks/foreman_webhooks.json
```

Wonder what made the template think that's not required

Reply:
Ah, your plugin doesn't have any asset pipeline assets (only webpack), so these paths are almost empty, but they exist...

Probably best to just put it back (and make the template include it `if has_assets || has_weback`)